modules/ROOT/pages/monitor-api-conformance.adoc

```diff
@@ -42,8 +42,22 @@ include::_partials/api-gov-concepts.adoc[tag=nonconformance-severity,leveloffset
 
 include::_partials/api-gov-concepts.adoc[tag=api-governance-usage-reports,leveloffset=+1]
 
-=== Governance Profile Report
-On the main API Governance console page, select the *Profiles* tab and then click the *Export Report* button to export a report of governance profile information, such as: 
+[[gov-conf-report]]
+== Governance Conformance Report
+
+You can view conformance details for governed APIs in their governance validation reports in API Governance console and API Manager. See xref:find-and-fix-conformance-issues.adoc[].
```

Comment:
(Global) Any way to remove "You can"? 
To view...., see...

Reply:
Yeah, I just realized with this edit that we prohibit "you can" in our standards. I also checked Microsoft, and apparently they call it "weak writing" now, so I'll need to change my habits. We should add this to Vale.